### PR TITLE
smart ads package

### PR DESCRIPTION
**Pull request by juanmaguitar**
As per #32

### CODE REVIEW

**Comment by cristianbote** [comment]
I would appreciate if we loop in @DAreRodz to give it an 👁️  as well

**Comment by juanmaguitar** [comment]
Sorry for all the irrelevant changes in the PR (gitbook’s fault)
The relevant file to check is `docs-api/frontity-packages/features-packages/smart-ads.md`

**Comment by juanmaguitar** [comment]
> I would appreciate if we loop in @DAreRodz to give it an 👁️ as well

Done! I've added @DAreRodz also as a reviewer 👍

File: docs-api/frontity-packages/features-packages/smart-ads.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+description: API reference of `@frontity/smart-adserver` package
+---
+
+# @frontity/smart-adserver
+
+Smart Adserver is an ad serving network and they provide an API for displaying the ads which is described [here](https://support.smartadserver.com/s/article/Tagging-guide)
+
+The `@frontity/smart-adserver` package will load a third-party Smart Adserver library which adds certain properties on the global `window` object. Then you as the developer can then use those properties to make the "ad call" (which are basically API calls to the Smart Adserver).
+
+In response, the Smart Adserver dynamically loads some code which will modify the DOM to insert the ad in the place that was specified in the API call.
```
Comment (DAreRodz) [inline docs-api/frontity-packages/features-packages/smart-ads.md]: Uhm, this is not completely rght. It's true that `@frontity/smart-adserver` loads a third-party library, but developers wouldn't have to interact with it. The one that uses the SmartAdserver API is the `<SmartAd>` component exposed in `libraries.fills.smartAdserver.SmartAd`.

I would write something like this instead:

```suggestion
The `@frontity/smart-adserver` package will load a third-party Smart Adserver library which adds certain properties on the global `window` object. Then you as the developer can render the exposed `<SmartAd>` component, which will use those properties to make the "ad call" (which are basically API calls to the Smart Adserver).

In response, the Smart Adserver dynamically loads some code which will modify the DOM to insert the ad in the place that the `<SmartAd>` component was rendered.
```

File: docs-api/frontity-packages/features-packages/smart-ads.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
+---
+description: API reference of `@frontity/smart-adserver` package
+---
+
+# @frontity/smart-adserver
+
+Smart Adserver is an ad serving network and they provide an API for displaying the ads which is described [here](https://support.smartadserver.com/s/article/Tagging-guide)
```
Comment (DAreRodz) [inline docs-api/frontity-packages/features-packages/smart-ads.md]: Just a missing `.` at the end of the sentence.

```suggestion
Smart Adserver is an ad serving network and they provide an API for displaying the ads which is described [here](https://support.smartadserver.com/s/article/Tagging-guide).
```

File: docs-api/frontity-packages/features-packages/smart-ads.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+| **`formatId`** | number | yes      | The `id` of the format as defined in [SmartAds](https://support.smartadserver.com/s/article/Ad-API-reference); represents an ad slot on a page (medium rectangle, skyscraper...); [_more info_](https://support.smartadserver.com/s/article/Setting-up-inventory) |
+| **`callType`** | string | yes      | The type of the ad call. Possible values: `iframe` or `std`                                                                                                                                                                                                       |
+| **`tagId`**    | string | no       | The `id` of the container that will contain the ad. Default Value: `sas_${formatId}"`                                                                                                                                                                             |
+
+| **`width`** | number | no | The width of the ad. Used with callType `iframe`. |
```
Comment (DAreRodz) [inline docs-api/frontity-packages/features-packages/smart-ads.md]: The table here is broken, there is an new line character that should be removed (line 91).

```suggestion
| **`tagId`**    | string | no       | The `id` of the container that will contain the ad. Default Value: `sas_${formatId}"`                                                                                                                                                                             |
| **`width`** | number | no | The width of the ad. Used with callType `iframe`. |
```

File: docs-api/frontity-packages/features-packages/smart-ads.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+Smart Adserver is an ad serving network and they provide an API for displaying the ads which is described [here](https://support.smartadserver.com/s/article/Tagging-guide)
+
+The `@frontity/smart-adserver` package will load a third-party Smart Adserver library which adds certain properties on the global `window` object. Then you as the developer can then use those properties to make the "ad call" (which are basically API calls to the Smart Adserver).
+
+In response, the Smart Adserver dynamically loads some code which will modify the DOM to insert the ad in the place that was specified in the API call.
+
+The package has 3 main components:
+
+- _The "Root" component_. It includes the `<Head>` that loads the Smart Adserver library. When the user adds the `@frontity/smart-adserver` to their `frontity.settings.js` file, this library will be loaded automatically.
+- _The `SmartAd` component_. This component is exposed in `libraries.fills.SmartAd`. The users can just use this component directly to display ads by passing it relevant props. The component takes care of calling the Smart Adserver API and injecting the ad into the DOM in the relevant place
```
Comment (DAreRodz) [inline docs-api/frontity-packages/features-packages/smart-ads.md]: The path where the component is exposed is wrong:

```suggestion
- _The `SmartAd` component_. This component is exposed in `libraries.fills.SmartAdserver.SmartAd`. The users can just use this component directly to display ads by passing it relevant props. The component takes care of calling the Smart Adserver API and injecting the ad into the DOM in the relevant place
```

**Review by DAreRodz — CHANGES_REQUESTED** [review]
LGTM, although there are some minor changes to do. I've added some suggestions: :point_down:

**Comment by juanmaguitar** [comment]
@DAreRodz I've applied your suggested changes.
Please, have another look when you can

**Review by DAreRodz — APPROVED** [review]
Great, thanks @juanmaguitar! :raised_hands:

**Pull request merged**